Automattic/jetpack
# Gardening action: add new Flag OSS feature

#### Changes proposed in this Pull Request:

This PR brings an additional task to the Gardening action, based on an existing feature currently enabled for the Calypso repo:

This new action flags PRs opened by external contributors (not from a branch on the main repo), adds the "OSS Citizen" label, and posts a Slack message to your team channel to let everyone know you've got a contribution from an external contributor.

**Note**: I chose to look at how they worked on their PR (from a fork) instead o
f looking at the organization listed in their profile to decide whether they
 would be considered as outside contributors.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Ensure the existing action features (auto-labeling, comment below this PR) still work for this PR.
* Since this runs on `pull_request_target`, we will not be able to test much before this gets merged. Then once this is merged, I'll open a test PR from a fork to test things in the #jeherve-debug channel.
